ODIN_II/SRC/netlist_create_from_ast.cpp

```diff
@@ -3354,6 +3354,17 @@ void terminate_continuous_assignment(ast_node_t* node, signal_list_t* assignment
                 warning_message(NETLIST, node->loc,
                                 "Assignment (%s) is missing driver\n", pin->name);
             } else {
+                /* For the check - Wire cannot be on left in always block */
+                if (node->type == ALWAYS) {
```

Comment:
In this function, we terminate the assignment, so it is not a good place to catch this error.

Reply:
Indeed, the Odin should not even proceed to this function when there is a non-blocking statement in always block.

Reply:
As I said the best place is alias_assignment since we alias signal to its driver in that function

Reply:
Do you mean assignment_alias?